test/MUXControlsTestApp/MainPage.xaml.cs

```diff
@@ -211,5 +224,19 @@ private void LongAnimationsDisabled_Unchecked(object sender, RoutedEventArgs e)
         {
             MUXControlsTestApp.App.DisableLongAnimations = false;
         }
+
+        private void FlowDirectionChooser_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            FrameworkElement root = Window.Current.Content as FrameworkElement;
```

Comment:
>Window.Current.Content [](start = 36, length = 22)

Window.Current.Content isn't used by islands - isn't this code shared with tests for win32 islands?

Reply:
No, not yet.